package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
All frontend-related stuff is contained under `/frontend` directory. You should incorporate your proposed changes there.

Reply:
if I want to install husky in our project we should initialize the .husky folder where the .github folder is contained(root folder) at the same time we need to install dependencies that is the reason I have created the package.json file in the root folder sir @undera